src/v/cluster/partition_balancer_types.h

```diff
@@ -133,13 +132,16 @@ operator<<(std::ostream& os, partition_balancer_status status) {
 
 struct partition_balancer_overview_request
   : serde::envelope<partition_balancer_overview_request, serde::version<0>> {
+    using rpc_adl_exempt = std::true_type;
```

Comment:
this is great 👍🏼

Reply:
hey lookie there!

Reply:
just to make sure, @ztlpn, feature manager is used to ensure that no adl-exempt / serde-only message are sent/received until all nodes have been upgraded?